### PR TITLE
Further updates to the README

### PR DESCRIPTION
**Pull request by davehunt**
Building on the README changes recently made by @rbillings, here are a few more tweaks to streamline this one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,10 @@ This repository contains tests for Mozilla's [Bouncer website](https://wiki.mozi
 
 ## Getting involved
 We love working with contributors to fill out the test coverage for Mozilla's
-support website, but it does require a few skills. By contributing to our test
+Bouncer, but it does require a few skills. By contributing to our test
 suite you will have an opportunity to learn and/or improve your skills with
 Python, GitHub, and virtual environments.
 
-For some resources for learning about these technologies, take a look at our
-documentation on [running Web QA automated tests][running-tests].
```
Comment (davehunt) [inline README.md]: This document is more about the Selenium tests, so I think it's likely to cause confusion here.

File: README.md [hunk 2]
```diff
@@ -27,9 +24,6 @@ in irc.mozilla.org [#mozwebqa][irc] to ask questions about our tests. We also
 have a [mailing list][list] available that you are welcome to join and post to.
 
 ## How to run the tests locally
-We maintain a [detailed guide][running-tests] to running our automated tests.
-However, if you want to get started quickly, you can try following the steps
-below:
```
Comment (davehunt) [inline README.md]: As above, I think this guide is less appropriate for bouncer.

File: README.md [hunk 3]
```diff
@@ -58,66 +52,22 @@ $ pip install -Ur requirements.txt
 ```
 
 ### Running tests locally
-
 To run these tests, use:
 
 ```bash
-py.test --baseurl="http://bouncer-bouncer.stage.mozaws.net" tests
+$ py.test --baseurl http://bouncer-bouncer.stage.mozaws.net
```
Comment (davehunt) [inline README.md]: No need for the tests directory, as pytest discovers them itself.

File: README.md [hunk 4]
```diff
@@ -58,66 +52,22 @@ $ pip install -Ur requirements.txt
...
 ```
 
 Use `-k` to run a specific test. For example,
 
 ```bash
-py.test -k test_that_checks_redirect_using_incorrect_query_values \
-        --baseurl="http://bouncer-bouncer.stage.mozaws.net" tests
-```
-
-The mozwebqa plugin has advanced command line options for reporting and using browsers. To see the options available, try running:
```
Comment (davehunt) [inline README.md]: These tests don't use the mozwebqa plugin.

File: README.md [hunk 5]
```diff
@@ -58,66 +52,22 @@ $ pip install -Ur requirements.txt
...
-
-```bash
-py.test --help
+$ py.test --baseurl http://bouncer-bouncer.stage.mozaws.net -k test_that_checks_redirect_using_incorrect_query_values
 ```
 
-Also see the documentation on davehunt's [pytest-mozwebqa](https://github.com/davehunt/pytest-mozwebqa) GitHub project page.
```
Comment (davehunt) [inline README.md]: As above, these tests do not use the mozwebqa plugin.

File: README.md [hunk 6]
```diff
@@ -58,66 +52,22 @@ $ pip install -Ur requirements.txt
...
-
-### `virtualenv` and `virtualenvwrapper` (optional, intermediate level)
```
Comment (davehunt) [inline README.md]: We link to a virtualenv doc earlier in the README now.

File: README.md [hunk 7]
```diff
@@ -58,66 +52,22 @@ $ pip install -Ur requirements.txt
...
-
-While most of us have had some experience using virtual machines, [`virtualenv`](https://pypi.python.org/pypi/virtualenv) is something else entirely. It's used to keep libraries that you install from clashing and messing up your local environment. After installing `virtualenv`, installing [`virtualenvwrapper`](https://bitbucket.org/dhellmann/virtualenvwrapper) will give you some nice commands to use with `virtualenv`.
-
-For a more detailed discussion of `virtualenv` and `virtualenvwrapper`, check out our [quick start guide](https://wiki.mozilla.org/QA/Execution/Web_Testing/Automation/Virtual_Environments) and also [this blog post](http://www.silverwareconsulting.com/index.cfm/2012/7/24/Getting-Started-with-virtualenv-and-virtualenvwrapper-in-Python).
-
-### Moz-Grid-Config (optional, intermediate level)
```
Comment (davehunt) [inline README.md]: No idea why this was in here, it doesn't use Selenium at all.

**Comment by m8ttyB** [comment]
r+

**Comment by rbillings** [comment]
Agree this looks more like the template, r+ once the branch is up to date.

**Pull request merged**